onnxruntime/core/framework/graph_partitioner.cc

```diff
@@ -848,20 +854,39 @@ static Status CreateEpContextModel(const ExecutionProviders& execution_providers
     }
   }
 
-  size_t ini_size_threshold = 0;
-  std::filesystem::path external_ini_path;
-  if (ep_context_ext_ini_path.empty()) {
+  size_t ini_size_threshold = ep_context_gen_options.output_external_initializer_size_threshold;
```

Comment:
Does anything ensure there's a sensible minimum? I believe 128 bytes is used in a few places. Fewer than that and it costs you more to store it in the external file than it it's worth as you need the filename, offset and length as well as the data.

Reply:
I set the default to 0 to preserve the current default behavior on the main branch.

Reply:
The problem with not respecting 128 threshold is that ONNX TypeAndShape inference functions sometimes request  the full body of the weight, and it does not know how to read our files. The ones it requests happen to be below 128 length.